src/v/cluster/topic_properties.cc

```diff
@@ -67,7 +68,8 @@ std::ostream& operator<<(std::ostream& o, const topic_properties& properties) {
       properties.mpx_virtual_cluster_id,
       properties.write_caching,
       properties.flush_ms,
-      properties.flush_bytes);
+      properties.flush_bytes,
+      properties.remote_label);
```

Comment:
Do `adl<cluster::topic_properties>::to()` and `adl<cluster::topic_properties>::from()` also need updating with the added `remote_label` property?

Reply:
No, the adl code is a (I think dead) remnant of a different serialization library we used before serde. We've stopped maintaining parity, so new fields are missing from it